app/src/main/java/com/example/githubusers/model/GithubUsersResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.example.githubusers.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+
+public class GithubUsersResponse {
+
+    @SerializedName("result")
+    private List<GithubUsers> result;
```

Comment:
This field `result` should describe what it contains

Reply:
Totally agree. I found it is supposed to be `items`